src/helpers/transformUniqueAssetTraitsForPresentation.ts

```diff
@@ -0,0 +1,55 @@
+import { format } from 'date-fns';
+import { UniqueAssetTrait } from '../entities/uniqueAssets';
+
+type MappedTrait = UniqueAssetTrait & {
+  originalValue: string | number;
+  keepLowerCase?: boolean;
+  disableMenu?: boolean;
+};
+
+/**
+ * Mapper function that converts various NFT trait values according to their
+ * data type and other qualities. Like if it's a link it is shortening it.
+ */
+export default function transformUniqueAssetTraitsForPresentation(
+  trait: UniqueAssetTrait
+): MappedTrait {
+  const { display_type, value } = trait;
+
+  if (display_type === 'date') {
+    // the value is in seconds, formatted like Jan 29th, 2022
+    const newValue =
+      typeof value === 'number' ? format(value * 1000, 'MMM do, y') : value;
+
+    return {
+      ...trait,
+      disableMenu: true,
+      originalValue: value,
+      value: newValue,
+    };
+  }
+
+  if (display_type === 'boost_percentage') {
+    return { ...trait, originalValue: value, value: `+${value}%` };
+  }
+
+  if (display_type === 'boost_number') {
+    return { ...trait, originalValue: value, value: `+${value}` };
+  }
+
+  if (
+    typeof value === 'string' &&
+    value.toLocaleLowerCase().startsWith('https://')
+  ) {
+    const newValue = value.toLowerCase().replace('https://', '');
```

Comment:
I don't mind this. I would merge this and if someone will complain, we can remove this  ¯\_(ツ)_/¯ . 

Also, the URL should be pressable (and open the URL). Can you include this in the PR? I guess we should `url?: string` to `MappedTrait` and then use this in a Context menu

Reply:
Or just check in `Tag` if the original value is an URL and defer logic to there

Reply:
Yeah, good idea with opening the URL I will make sure to include it as well

Reply:
Actually I think there's more to this. The context menu says sth like "search for this trait on OpenSea" I think we should entirely change the context menu there, like to include open the URL option as well. We can make this a lot earlier I guess.

Reply:
```suggestion
    const newValue = value.toLowerCase().substring(8);
```
This is what we actually want to do no?